tests/sdk.test.ts

```diff
@@ -12,8 +13,11 @@ const account = "0x3f5ce5fbfe3e9af3971dd833d26ba9b5c936f0be";
 beforeAll(async () => {
   const web3 = await new Web3(new Web3.providers.HttpProvider(process.env.INFURA_URL_HTTP || ""));
   const provider: ExternalProvider = (new Web3Provider(web3.currentProvider as any)).provider;
+  const ethersProvider = new ethers.providers.JsonRpcProvider(process.env.INFURA_URL_HTTP || "");
+  /// @notice Passing ethersProvider is just for apr integration on v2
   SDK = await new Degenerative({
     provider: provider,
+    ethersProvider: ethersProvider,
```

Comment:
we dont need to have 2 different providers but only 1 provider which should work despite how it looks like, you can convert providers if that is nessesary after passing it through `provider`